local-helm/local-overrides.yaml

```diff
@@ -8,6 +8,7 @@
 #     repository: nhsd/buying-catalogue/document-dapi
 
 dbPassword: DisruptTheMarket1!
```

Comment:
Can both these top level value settings also go in buyingcatalogue/values.yaml  please